lte/gateway/docker/services/python/Dockerfile

```diff
@@ -104,8 +104,6 @@ RUN apt-get update && apt-get install -y \
   wget \
   ethtool \
   linux-headers-generic \
-  bpfcc-tools \
-  python3-bpfcc \
```

Comment:
Given that the rest of this file wasn't changed, why is this change necessary?

Reply:
Ah I see, just read it in the PR description.

Reply:
@m-govind You originally added those lines in 6181da05. Do you see a problem here?